test/CMakeLists.txt

```diff
@@ -78,13 +78,15 @@ add_fmt_test(printf-test)
 add_fmt_test(ranges-test ranges-odr-test.cc)
 add_fmt_test(scan-test)
 add_fmt_test(std-test)
+add_fmt_test(ranges-std-test)
```

Comment:
Please merge this into `std-test`. Having a separate binary for a single test case is an overkill and including `fmt/ranges.h` from std-test is fine.

Reply:
Are you really sure that you want that? This means that your preexisting test does not actually test anymore that the inclusion of stl.h suffices to get the wanted effects, it only tests that the combined inclusion has these effects. I really recommend to keep the separate test translation unit, because this introduces exactly the errornous situation.

Reply:
> Are you really sure that you want that?

Yes. Just make sure to keep `fmt/std.h` as the first include. I'm not concerned about other interactions between headers.

Reply:
Done.